modules/tabs-v2/assets/scss/frontend.scss

```diff
@@ -154,6 +190,30 @@
 
 		display: var(--tabs-v2-wrapper-display);
 		justify-content: var(--tabs-v2-wrapper-justify-content);
+
+		.elementor-tab-title {
+			.elementor-icon-active {
+				display: none;
+		  	}
+		}
+		.elementor-active {
+			.elementor-icon-active {
+				display: unset;
+			}
+			.elementor-icon-selected {
+		  		display: none;
```

Comment:
Why hide them?

Reply:
when its not active (".elementor-tab-title .elementor-icon-active") => the  active icon is hidden 
when its active (".elementor-active") => the  active icon is unset and the selected ( the original ) is hidden 
@matipojo